vuepress/docs/next/docs/component-generator/component-gen-tech.md

```diff
@@ -7,7 +7,7 @@ When generating a microservice using the Entando Component Generator there are a
 * Microservice code
   * The Component Generator currently only supports the generation of Spring Boot microservices but will support the generation of Quarkus based services soon. 
   * If you're interested in using Quarkus in your Entando app, contact us on the [Forum](https://forum.entando.org/) or [Community Slack](https://join.slack.com/t/entandocommunity/shared_invite/zt-g609owdv-2K~YRh8zrI6lqlWo4aFWUw) and we'll help you get started.
-  * If you're considering non-Java based microservices, [here is a definition of an Entando plugin](../ecr/ecr-bundle-details.html#plugin), including information on the runtime contract required to use them. 
+  * If you're considering non-Java based microservices, [here is a definition of an Entando plugin](../ecr/ecr-bundle-details.md#plugin), including information on the runtime contract required to use them. 
```

Comment:
Sorry about these html issues. I did change all of these in the Customer Portal PR when I saw them but should have separated  the PR out since that one is taking a while to complete. I'll remember it for next time since I'm writing this.